Resources/Prototypes/Reagents/medicine.yml

```diff
@@ -77,6 +77,21 @@
           groups:
             Brute: -2
 
+- type: reagent
+  id: Codeine
+  name: codeine
```

Comment:
Do we have any other references to actual drugs? If so they should be culled.

Reply:
Meth, Marijuana, Alcohol, Space Drugs, uhhhh, alot more i think

Reply:
Nicotine

Reply:
Codeine is usually the main active ingredient in prescription cough medicine

Reply:
All of those are restricted to varying degrees so it's a bit weird.